generator/tree.go

```diff
@@ -42,7 +44,7 @@ func walkNode(n *Node, f func(n *Node)) {
 }
 
 // Transform the tree.
-func prepareTree(nodes *Node) map[string]*Node {
+func prepareTree(nodes *Node, logger log.Logger) (map[string]*Node, error) {
```

Comment:
I don't see where this `error` return is non-nil in this function.

Reply:
Line 148

Reply:
Line 148 is a `nil` return.

Reply:
Sorry, I'm not awake yet. Fixed.